src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1938,7 +1938,8 @@ fn resolution_failure(
                             | Use
                             | LifetimeParam
                             | Ctor(_, _)
-                            | AnonConst => {
+                            | AnonConst
+                            | InlineConst => {
```

Comment:
These should probably just be `unreachable!` ... but this change is fine, it should never be hit in practice.